stories/dynamic-page/dynamic-page.stories.js

```diff
@@ -0,0 +1,1004 @@
+                <ul class="fd-tabs fd-tabs--l fd-dynamic-page__tabs fd-dynamic-page__tabs--sm fd-dynamic-page__tabs--add-shadow" role="tablist">
+                    <li role="listitem" class="fd-tabs__item">
+                        <a class="fd-tabs__link" aria-controls="fuCwV553" href="#fuCwV553" role="tab">
+                            <span class="fd-tabs__tag">
+                                Link
+                            </span>
+                        </a>
+                    </li>
+                    <li role="listitem" class="fd-tabs__item">
+                        <a class="fd-tabs__link" aria-controls="AiWfz168" aria-selected="true" href="#AiWfz168" role="tab">
+                            <span class="fd-tabs__tag">
+                                Selected
+                            </span>
+                        </a>
+                    </li>
+                </ul>   
+            </div>
+        </header>
+        <div class="fd-dynamic-page__content fd-dynamic-page__content--sm">
+            <div class="fd-tabs__panel" aria-expanded="false" id="fuCwV553" role="tabpanel">
+                Lorem ipsum dolor, sit amet consectetur adipisicing elit. Rerum illo voluptatem, unde voluptate aliquam 
+                eum tempore aliquid dolorem cumque ullam perspiciatis omnis et asperiores dolores, consequatur, suscipit sint animi amet?
+                Lorem ipsum dolor, sit amet consectetur adipisicing elit. Rerum illo voluptatem, unde voluptate aliquam eum tempore aliquid 
+                dolorem cumque ullam perspiciatis omnis et asperiores dolores, consequatur, suscipit sint animi amet?
+            </div>
+            <div class="fd-tabs__panel" aria-expanded="true" id="AiWfz168" role="tabpanel">
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+            </div>
+        </div>
+        <footer class="fd-bar fd-bar--floating-footer" style="position:relative">
+            <div role="toolbar" aria-label="Finalizing actions" class="fd-bar__right">
+                <div class="fd-bar__element">
+                    <button aria-label="button" class="fd-button fd-button--emphasized fd-button--compact">Save</button>
+                </div>
+                <div class="fd-bar__element">
+                    <button aria-label="button" class="fd-button fd-button--transparent fd-button--compact">Cancel</button>
+                </div>
+            </div>
+        </footer>
+    </section>
+</div>
+`;
+
+mobileCollapsed.storyName = 'Collapsed Header in Mobile';
+
```

Comment:
Since you've already described the mobile dynamic page, I would keep it really simple:

In the example below, the dynamic page is displayed with a collapsed header for mobile screens.